iroh-bitswap/src/lib.rs

```diff
@@ -670,21 +659,16 @@ mod tests {
     }
 
     fn mk_transport() -> (PeerId, Boxed<(PeerId, StreamMuxerBox)>) {
-        let local_key = Keypair::generate_ed25519();
+        let local_key = libp2p_identity::Keypair::generate_ed25519();
```

Comment:
Do we try to persist this key anywhere or will we get a new key at each boot? Can it be passed in in the future?